src/Components/App/App.js

```diff
@@ -67,41 +60,79 @@ class App extends Component {
       .catch(error => {
         console.log('Movies Request Failed', error)
         const filteredResponses = responseStatus.filter(status => status > 299)
-        this.setState({ error: error, errorStatus: Number(filteredResponses) })
+        this.setState({ error: error, errorStatus: Number(filteredResponses), isLoading: false })
+      })
+  }
+  
+  triggerDropDown = () => {
+    this.setState({ triggerDropDown: true })
+  }
+
+  sortByRatings = () => {
+    const sortedMovies = this.state.movies.sort((a,b) => {
+      return b.average_rating - a.average_rating
     })
+    return sortedMovies
+  }
```

Comment:
This looks good! To be picky we could also return the sorting straight away without assigning it to a variable, but that's just my default.